### ISSUE TITLE
[UX Issue/Request] Have contributions / project pages have a callout/banner to contributors.

### ISSUE THREAD

**Issue by ryantheleach** [utopian-io/utopian.io#369]
**TLDR:**
utopian.io is suffering due to a lack of uniform UI with join.utopian.io and utopian.io/rules

As a brand new user, stumbling onto the main site, where the guts of stuff happens, I literally have no idea what is going on. I'm quickly turned off by the lack of information, on top of the existing prejudice due to *strange* activity on our projects lately.

I would like to see the information better linked, better presented, and a uniform header created across join.utopian.io utopian.io/rules and utopian.io

**User Story:** Point of View of an open source collaborator.

Recently we have been seeing translations being posted to our Github Repo (We accept them via CrowdIn)  and also Translators seemingly appear out of no where for our Documentation.

We love to see the visibility that this platform is getting us!

However we were left very confused. Where were the people coming from? Why did some of the translations appear to be machine translated? These people sometimes provided quality translations, but appeared to have no working knowledge of our project.

Eventually one of them mentioned `needing` proof-reader in order for their translations to count on utopian.io

Google searching our project and utopian.io or following the link to their contribution, lands us at this page: (This is not a comment on the quality of this authors work, I do not know Spanish and have not reviewed it)
https://utopian.io/utopian-io/@fucho/minecraft-modding-platform-spongepowered-translation

However, this page lacks ANY useful information or callouts or banners. I'm left exploring Utopian.io which, was entirely void of any helpful information.

Eventually I stumbled across a moderator comment, linking to a discord, and after a chat with the moderators, found useful information at  https://join.utopian.io/ and https://utopian.io/rules  Which appears to have an entirely different UI then the main interface.

All this confusion would have been avoided, and a potential project maintainer converted, had this information been targeted at project maintainers.

**Comment by olo2552** [comment]
It should be resolved in the new UI, but it is something worth noticing during the new UI's review.
